protocol-units/settlement/mcr/client/src/lib.rs

```diff
@@ -4,14 +4,16 @@ use tokio_stream::Stream;
 #[cfg(test)]
 pub mod tests;
 
-#[cfg(feature = "mock")]
 pub mod mock;
 
-// #[cfg(feature = "eth")]
-// pub mod eth;
-#[cfg(feature = "eth")]
+#[cfg(feature = "mock")]
+pub use mock::*;
```

Comment:
This is also shared with #191, it would be better to stage these branches for incremental reviews.

Reply:
Will do.